src/components/Button/buttonStyles.js

```diff
@@ -17,9 +17,38 @@ const defaultStyle = props => css`
     })}
   background: ${props => props.theme.colors.button.background};
   color: ${props => props.theme.colors.button.color};
+  ${props =>
+    !props.weight && `font-weight: ${props.theme.typography.fontMedium}`};
+
+  &:focus,
+  &:hover,
+  &:active {
+    ${props =>
+      !props.background &&
+      `background-color: ${props.theme.colors.button.activeBackground};`}
+    ${props =>
+      !props.border &&
+      `border-color: ${props.theme.colors.button.activeBorderColor};`}
+  }
+
```

Comment:
I'm not sure exactly where these are set but it looks like the focus styles are not quite right: we are setting the outline correctly but the colour of the font should not change and the background colour should be the same as the outline.

Reply:
not sure if I understand entirely but the font color doesn't change when focused and you're right, there's an outline missing when focused

Reply:
I'm looking at this for instance - seems like the outline's fine but the text has changed colour and there's no background. 
![image](https://user-images.githubusercontent.com/6080548/80083054-13d62a00-854d-11ea-8542-85e9897cacc3.png)

Reply:
That seems to be the anchor and not a button, let me investigate